libs/dh/esett/data-access-outgoing-messages/src/lib/dh-outgoing-messages.store.ts

```diff
@@ -112,7 +112,7 @@ export const DhOutgoingMessagesSignalStore = signalStore(
     ),
     loading: outgoingMessageQuery().loading,
     downloading: downloadDocumentQuery().loading,
-    hasError: computed(() => outgoingMessageQuery().error() !== undefined),
+    hasError: computed(() => outgoingMessageQuery().hasError()),
```

Comment:
```suggestion
    hasError: outgoingMessageQuery().hasError,
```

Reply:
outgoingMessageQuery is a signal as well, so i think it needs to be in computed